lib/common/filters/exponentialretrypolicyfilter.js

```diff
@@ -76,9 +76,8 @@ ExponentialRetryPolicyFilter.prototype.shouldRetry = function (statusCode, reque
   var incrementDelta = Math.pow(2, retryData.retryCount) - 1;
   var boundedRandDelta = this.retryInterval * 0.8 + Math.floor(Math.random() * (this.retryInterval * 1.2 - this.retryInterval * 0.8));
   incrementDelta *= boundedRandDelta;
-
+      
   retryData.retryInterval = Math.min(this.minRetryInterval + incrementDelta, this.maxRetryInterval);
-  retryData.retryable = retryData.retryCount ? retryData.retryCount < this.retryCount : true;
```

Comment:
If retry count is the hard bar of retry policies, can we just return false here when it exceeds?